### DISCUSSION TITLE
Nivo Production Ready?

### DISCUSSION THREAD

**Discussion by George-Botron** [plouc/nivo#1845] [Q&A]
Is Nivo as of the latest release version 0.73.1 production ready?
If it's not what is the ETA of production ready release?

**Comment by plouc** [comment]
Yes, it is, do you have specific concerns?

However, please note that until nivo reaches version 1, you should expect BC breaks, and I don't have an ETA for V1 due to how I maintain the project, during my spare time, without earning money for it, except a few sponsors, but this barely covers the time I spend reviewing issues, answering on discord, reviewing PRs (when I can).
